pkgs/tools/admin/eksctl/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "eksctl";
+  version = "0.1.35";
+
+  src = fetchFromGitHub {
+    owner = "weaveworks";
+    repo = "eksctl";
+    rev = version;
+    sha256 = "0b3s7vh85k68wawmsdp96q9l4yhikwhyjn1c7cwxys0aia4i8wkv";
+  };
+
+  goPackagePath = "github.com/weaveworks/eksctl";
+
+  buildPhase =
+    ''
+      runHook preBuild
+      runHook renameImports
+
+      cd go/src/$goPackagePath
+
+      go install \
+        -v \
+        -p $NIX_BUILD_CORES \
+        -tags "netgo release" \
+        -ldflags "-s -w" \
+        ./cmd/eksctl
+
+      runHook postBuild
+    '';
+
+  meta = with stdenv.lib; {
+    description = "A CLI for Amazon EKS";
+    homepage = https://github.com/weaveworks/eksctl;
```

Hint: We're probably going to deprecate unquotes URL syntax when RFC 45 is approved.

```suggestion
    homepage = "https://github.com/weaveworks/eksctl";
```